CMakeLists.txt

```diff
@@ -493,6 +509,7 @@ if(EMSCRIPTEN)
     target_link_libraries(binaryen_js "-sFILESYSTEM=1")
   endif()
   target_link_libraries(binaryen_js "-sNODERAWFS=0")
+  target_link_libraries(binaryen_js "-sSINGLE_FILE")
```

Comment:
Why do we need to specify this twice?  i.e. do we not have a place to specify flag for both _js and _wasm?

Reply:
We have some duplication here that we've never figured out a way to remove... I'm not sure if CMake has a way? We basically want to "inherit" one target from another.